go/core_docs.bzl

```diff
@@ -0,0 +1,135 @@
+"""
+  ["Make variable"]: https://docs.bazel.build/versions/master/be/make-variables.html
```

Comment:
nit: typically markdown authors put these hyperlinks at the end, not the beginning

Reply:
Right, I think I had these there before I added the footers. I'll move them over